Platforms/QemuSbsaPkg/VirtNorFlashStandaloneMm/VirtNorFlash.h

```diff
@@ -0,0 +1,346 @@
+/** @file  NorFlash.h

```

Comment:
Ideally, this should match the filename.

```suggestion
/** @file  VirtNorFlash.h

```

Reply:
Fixed.